lib/protocol/networks.js

```diff
@@ -480,6 +480,16 @@ main.deployments = {
     required: false,
     force: false
   },
+  airstop: {
+    name: 'airstop',
+    bit: 2,
+    startTime: 1751328000, // July 1st, 2025
+    timeout: 1759276800, // October 1st, 2025
```

Comment:
Given how the windows work, October 1st will be in the middle of the window. This will actually make that last window non-existent. This will essentially make 2025 Sep 23 the last day of voting.

Windows ends:
```
1 - Tue, 15 Jul 2025 00:00:00 GMT
2 - Tue, 29 Jul 2025 00:00:00 GMT
3 - Tue, 12 Aug 2025 00:00:00 GMT
4 - Tue, 26 Aug 2025 00:00:00 GMT
5 - Tue, 09 Sep 2025 00:00:00 GMT
6 - Tue, 23 Sep 2025 00:00:00 GMT
7 - Tue, 07 Oct 2025 00:00:00 GMT
8 - Tue, 21 Oct 2025 00:00:00 GMT
```

If we want to include October 1st, maybe do October 7 + 10 minutes (make it a day Oct 8).

 bip9 stats will prob incorrectly say that auction is still in `started` state in the last auction, but timeout check just skips the last period.

Reply:
makes sense, updated to 8th Oct 2025